src/lib/support/jsontlv/TlvJson.cpp

```diff
@@ -22,6 +22,25 @@
 #include <lib/support/jsontlv/TlvJson.h>
 
 namespace {
+
+constexpr uint32_t kTemporaryImplicitProfileId = 0x1122;
```

Comment:
Why are we changing the existing code here?

Reply:
Mass update picked this too. Reverted now. it seems quite anoying we have 2 codebases for this ... this should be removed (ut this is to be done separately I guess, to make sure no odd users out there).